service/installer/helm/helm.go

```diff
@@ -260,7 +269,7 @@ func (i *HelmInstaller) login() error {
 
 	return i.runHelmCommand(
 		"login",
-		"registry",
+		"quay",
```

Comment:
Even though its command name is `quay`, this is the same plugin command as `helm registry` from helm2. They only change the name to quay since `helm registry` is a reserved command from helm3.